### ISSUE TITLE
Tests Depend on a specific version XMLRPC::Lite.

### ISSUE THREAD

**Issue by bluefeet** [redhotpenguin/perl-XML-Parser-Lite#4]
Nothing in the META.json declares a dependency on XMLRPC::Lite, but some of the tests depend on it being a specific version.  You should modify the tests to not only check for XMLRPC::Lite being installed, but also of being of a version that supports the features you are using.

Here's what I get when I run the tests:

```
t/37-mod_xmlrpc.t ...... Undefined subroutine &XMLRPC::Test::Server::run_for called at t/37-mod_xmlrpc.t line 16.
# Looks like your test exited with 255 before it could output anything.
```

This is with XMLRPC::Lite version `0.714`.  When I upgrade to the latest, version `0.717` the test passes.

**Comment by paultcochrane** [comment]
It looks like this issue has already been resolved by e1332e6448ec38d34f9e9eaedfca38149c898068.  Is this issue still relevant in some way?

**Comment by bluefeet** [comment]
I can't imagine it wouldn't do anything but resolve this issue.  Thanks!
